pkg/cmos/cmos.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package cmos lets you read and write to cmos registers while doing basic checks on valid register selections.
+
+package cmos
+
+import (
+	"errors"
+	"github.com/u-root/u-root/pkg/memio"
+)
+
+func Read(reg int64, data memio.UintN) error {
+	regVal := memio.Uint8(reg)
```

Comment:
Also, use fmt.Errorf and, also, a message like "byte %d is inside the range 0-13 which is reserved for RTC" is certainly a bit easier to debug than ""bytes below 14 are reserved for RTC"